### PR TITLE
Translation settings

### PR DESCRIPTION
**Pull request by Mr-KayJayDee**

### CODE REVIEW

File: src/app/[locale]/(app)/settings/_components/Languages.tsx [hunk 1]
```diff
@@ -112,7 +112,7 @@ function LocaleSwitcher({
                     as="h3"
                     className="text-lg font-bold text-white"
                   >
-                    Switch language
+                    {t("settings.languages.switch")}
```
Comment (florian-lefebvre) [inline src/app/[locale]/(app)/settings/_components/Languages.tsx:115]: You need to import `t` in this component as well

**Review by florian-lefebvre — CHANGES_REQUESTED** [review]
Awesome! Your contribution is really clean 👌 I left a really small review

File: src/app/[locale]/(app)/settings/_components/Languages.tsx (hunk 1 above)
Comment (Mr-KayJayDee) [reply src/app/[locale]/(app)/settings/_components/Languages.tsx:115]: Thanks ! Yes, when repeating things I may forget this 😅

Comment (Mr-KayJayDee) [reply src/app/[locale]/(app)/settings/_components/Languages.tsx:115]: After looking at it it's already the case 🤔

Comment (Mr-KayJayDee) [reply src/app/[locale]/(app)/settings/_components/Languages.tsx:115]: https://github.com/dumpus-app/dumpus-app/blob/02a4d27cc34f86e6945d1aefbeb130e022d9ac0c/src/app/%5Blocale%5D/(app)/settings/_components/Languages.tsx#L6

**Pull request merged**